AEPMessaging/Tests/UnitTests/Event+MessagingTests.swift

```diff
@@ -203,13 +203,22 @@ class EventPlusMessagingTests: XCTestCase {
                      source: EventSource.requestContent, data: data)
     }
     
-    func getSetPushIdentifierEvent(overriddingData: [String: Any]? = nil) -> Event {
-        let data = overriddingData ?? [
+    private func getSetPushIdentifierEvent(overridingData: [String: Any]? = nil) -> Event {
+        let data = overridingData ?? [
```

Comment:
lol, your spellcheck is working fixing my typos now! 😆

Reply:
Its doing magic 😅